tests/test_image.py

```diff
@@ -2,6 +2,10 @@
 
 def test_png_lilypond():
     a = converter.parse('tests/sample.musicxml').write('lilypond.png')
-def test_png_mscore():   
-    #b = converter.parse('tests/sample.musicxml').write('musicxml.png')
-    pass
\ No newline at end of file
+def test_png_mscore_from_mxl_short():   
+    easy_task = converter.parse('tests/sample.musicxml').write('musicxml.png')
+def test_png_mscore_from_mxl_long(): 
+    long_task = converter.parse('tests/ActorPreludeSample.musicxml').write('musicxml.png')
+def test_png_mscore_from_midi():
+    long_midi_task = converter.parse('tests/Super_Mario_64.mid').write('musicxml.png')
```

Comment:
Testing is never sufficient, to say the least, but I encourage you to try as many files with different formats as possible so that you can catch bugs at development phase. Let me provide you a whole bunch of musicxml - you may want to go over all of them and choose to incorporate some to the test suites. 
[xmlsamples.zip](https://github.com/lucien386/MusicXML2ABCNotationWebApp/files/4977208/xmlsamples.zip)


There are some further thoughts that I have regarding expanding testing. Since you are extending this to different format, can you think about scheme like given a musicxml, and then change it to back, midi, etc. and then those converted files can be further used for testing.

Reply:
my thought on testing:
the way that converter works now makes it easier to test across different input formats. As long as it is a valid format (as I have set in the list `ALLOWED_EXTENSIONS`), it will first be processed by `music21` as it tries to convert the content to a `music21.Score` object. Here's what's good about it: if the file is corrupted/has invalid content, no `music21.Score` object will be generated.

So here's how I plan to write tests: 

For `musicxml` format, I simply imitate uploading every single musicxml files in `xmlsample.zip`, and each of them should generate a `Score`. I will then upload a corrupted file for `musicxml`, and the app should generate nothing.

Reply:
Then I will extend this testing method to all other valid upload format, where I upload 3-5 valid file and 1 invalid file for each.

Reply:
Same on text submission

Reply:
For output, I will take a long musicxml and store it as a `music21.Score`, and I will execute `music21.converter.write` for every supported output format once and test for download.

Reply:
This should conclude the entire testing process.